aws/structure.go

```diff
@@ -955,10 +955,25 @@ func flattenNeptuneParameters(list []*neptune.Parameter) []map[string]interface{
 }
 
 // Flattens an array of Parameters into a []map[string]interface{}
-func flattenDocDBParameters(list []*docdb.Parameter) []map[string]interface{} {
+func flattenDocDBParameters(list []*docdb.Parameter, d *schema.ResourceData) []map[string]interface{} {
```

Comment:
Nit: Recommend passing in only the necessary information, rather than the entire `schema.ResourceData`, e.g.

```suggestion
func flattenDocDBParameters(list []*docdb.Parameter, configParameters []interface{}) []map[string]interface{} {
```